linkerd/metadata.csv

```diff
@@ -38,3 +39,31 @@ linkerd.rt.client.request_latency_ms.quantile,gauge,,millisecond,,Stats of the l
 linkerd.rt.server.connections,gauge,,connection,,Number of active connections for the server,0,linkerd,server active connections
 linkerd.rt.server.connects_s,rate,,connection,second,Number of connection par second for the server,0,linkerd,server connects/s
 linkerd.rt.server.request_latency_ms.quantile,gauge,,millisecond,,Stats of the latency of requests in milliseconds for the server,-1,linkerd,server requests latency
+linkerd.request_total,count,,,,Total count of HTTP requests.,0,linkerd,
```

Comment:
Should we add to the description a note about the version required for each metric ? e.g.

```
linkerd.response_latency,gauge,,millisecond,,Elapsed times between a request's headers being received and its response stream completing (require Linkerd 2),0,linkerd,
```

Reply:
Example: https://github.com/DataDog/integrations-core/pull/3105

Reply:
Done